### PR TITLE
Fix: embed.FS filepath that follow the unix style file path when running on windows

### PR DESCRIPTION
**Pull request by clarechu**
Signed-off-by: clarechu [1062186165@qq.com](mailto:1062186165@qq.com)

### Description of your changes

Fixes #

In Windows 10， When running vela-core

```log
"msg"="Reconciler error" "error"="object level reconcile error, type: \"Workflow\", msg: \"open static\\\\builtin-apply-compon
ent.cue: file does not exist\"
```

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by clarechu** [comment]
```go
filepath.Join(templateDir, file.Name()) ==> (static\builtin-apply-component.cue)

```

image

File: pkg/workflow/tasks/template/load.go [hunk 1]
```diff
@@ -60,7 +60,8 @@ func (loader *WorkflowStepLoader) LoadTaskTemplate(ctx context.Context, name str
 	staticFilename := name + ".cue"
 	for _, file := range files {
 		if staticFilename == file.Name() {
-			content, err := templateFS.ReadFile(filepath.Join(templateDir, file.Name()))
+			fileName := fmt.Sprintf("%s/%s", templateDir, file.Name())
+			content, err := templateFS.ReadFile(fileName)
```
Comment (wonderflow) [inline pkg/workflow/tasks/template/load.go:64]: this is interesting, the embed filesystem still follow the unix style file path when running on windows.

**Review by wonderflow — APPROVED** [review]

File: pkg/workflow/tasks/template/load.go (hunk 1 above)
Comment (clarechu) [reply pkg/workflow/tasks/template/load.go:64]: tks @wonderflow

**Review by FogDong — APPROVED** [review]

**Pull request merged**